setup/quickstart/halyard-gke-deploy-rbac/index.md

```diff
@@ -68,36 +68,58 @@ At the end of this guide you will have:
    gcloud projects add-iam-policy-binding $GCP_TEST \
      --role roles/container.admin \
      --member serviceAccount:$HALYARD_SA_EMAIL
-   ``` 
+   ```
 
-## Part 3: Add a service account (SA) to Kubernetes
+## Part 3: Add a Configure Kubernetes roles (RBAC) to Kubernetes
 
-1. Next we need to add a service account to Kubernetes that will handle the authorization inside the Kubernetes cluster.
-   Create a file, spinnaker-service-account.yaml, with the following content:
+1. Next we need to add a RBAC to Kubernetes that will handle the authorization inside the Kubernetes cluster.
+   Create a file, spinnaker-rbac.yaml, with the following content:
 
    ```yaml
+   apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Given that this is a just a demo setup -- what do you think of giving the cluster role binding "cluster-admin" for simplicity? We can still link to this RBAC spec if someone is interested in more granular permissions.

Reply:
I agree with you.
The cluster role binding "cluster-admin" is the most simple setting.
For someone who want to know how to configure RBAC, I will add a link for the RBAC specification.